### PR TITLE
Fixes an NullPointerException thrown in the video constraints compatibility module.

### PR DESCRIPTION
**Pull request by gpolitis**

### CODE REVIEW

**Comment by gpolitis** [comment]
strange that the tests fail on this, I'll look into it tomorrow

File: src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java [hunk 1]
```diff
@@ -470,6 +470,10 @@ public BridgeChannelMessage selectedEndpoints(SelectedEndpointsMessage message)
     {
         Set<String> newSelectedEndpoints = new HashSet<>(message.getSelectedEndpoints());
 
+        if (logger.isDebugEnabled())
+        {
+            logger.debug("Selected " + newSelectedEndpoints);
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java:475]: You can now do `logger.debug(() -> "Selected " + newSelectedEndpoints);` and avoid the extra `if` check.

File: src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java [hunk 2]
```diff
@@ -488,6 +492,12 @@ public void setSenderVideoConstraints(
         // Don't "pollute" the video constraints map with constraints for this
         // endpoint.
         videoConstraintsMap.remove(endpoint.getID());
+
+        if (logger.isDebugEnabled())
+        {
+            logger.debug("New video constraints map: " + videoConstraintsMap);
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java:498]: Same as above

File: src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java (hunk 1 above)
Comment (gpolitis) [reply src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java:475]: Doesn't this particular syntax create an anonymous class type (once) and a new instance every time it runs? Makes me wonder if simply concatenating the two strings is more efficient.

**Pull request merged**